src/bp/core/misc/utils.ts

```diff
@@ -84,3 +84,12 @@ export const asBytes = (size: string) => {
 
   return Number(matches[1])
 }
+
+export const bytesToString = (bytes: number): string => {
+  const units = ['bytes', 'kb', 'mb', 'gb', 'tb']
+  const power = Math.log2(bytes)
+  const unitNumber = Math.min(Math.floor(power / 10), 4)
+  const mantisse = bytes / Math.pow(2, unitNumber * 10)
```

Comment:
seems like you're using english everywhere except for 'mantisse'. From wiki, Significand would be the english term for it

Reply:
I copied that piece of code from the language page on the admin UI, i'm not even sure what mantisse means, but I can sure change it for your suggestion :-P

Reply:
I think I'm the one who wrote it at first ><